src/spec/help-spec.ts

```diff
@@ -27,13 +27,34 @@ beforeAll(() => {
 	});
 })
 
+function createInputMessage(text: string) {
+	return { personId: 'person', text: text};
```

Comment:
whitespace before closing brace

Reply:
I'll fix it, but you really are very pedantic.

Reply:
Style matters! This looks so ugly. :(

But in seriousness, I'd let this slide if I didn't already have a couple of more significant change recommendations. And then just fix it myself somewhere down the line.

Reply:
Fixed now :)